### ISSUE TITLE
Brownie Install Fails

### ISSUE THREAD

**Issue by iyushaw** [eth-brownie/brownie#1704]
Hello,

I am trying to install brownie on ubuntu 23.04 and I keep getting this error. All solutions on the forum and googling seem not to work. Below is the error am getting:

```
user$ pipx install eth-brownie
Fatal error from pip prevented installation. Full pip output in file:
    /home/pipi/.local/pipx/logs/cmd_2023-08-12_23.13.34_pip_errors.log

pip seemed to fail to build package:
    cytoolz==0.12.0

Some possibly relevant errors from pip install:
    error: subprocess-exited-with-error
    InternalError: Internal compiler error: Previous declaration is here
    Cython.Compiler.Errors.CompileError: cytoolz/functoolz.pyx

Error installing eth-brownie.
```
Environment:
Python 3.11.4
pipx 1.1.0
Linux: Linux 14-IIL 6.2.0-27-generic #28-Ubuntu SMP PREEMPT_DYNAMIC Wed Jul 12 22:39:51 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=23.04
DISTRIB_CODENAME=lunar
DISTRIB_DESCRIPTION="Ubuntu 23.04"

**Comment by liam-ot** [comment]
Same issue on MacOS

Python 3.11.3
pipx 1.2.0
attempting to install latest version of eth-brownie 1.19.3

I have this issue open on the cytoolz [repo](https://github.com/pytoolz/cytoolz/issues/202) which may be helpful

**Comment by Husienvora** [comment]
facing same issue

**Comment by iamdefinitelyahuman** [comment]
Issues installing with python `>=3.10` are finally fixed with `v1.20.0`.  However, note that I no longer actively maintain this repo.
